src/components/overview/OverviewMain.jsx

```diff
@@ -261,10 +366,26 @@ class overviewMain extends React.Component {
 
   //submit add to cart on button click
   addToCart() {
-    //need to create API request
-    console.log('this.cart', this.state.cart);
+    //API request does nothing
```

Comment:
A lot of added code in this file. Possibly think about ripping out some of the helper functionality into separate files.